src/Annotation.cpp

```diff
@@ -500,4 +510,16 @@ Annotation *generateAnnotation(const std::string &rawAnnotationString) {
   }
 
   return new ErrorAnnotation();
-}
\ No newline at end of file
+}
+
+std::string annotationTypeToString(AnnotationType anno) {
+  switch (anno) {
+  case AnnotationType::MustCallAnnotation:
+    return "MustCallAnnotation";
+    break;
+  case AnnotationType::CallsAnnotation:
+    return "MustCallAnnotation";
```

Comment:
This doesn't seem right (copy-paste mistake?)

Reply:
Yep; this'll be fixed